packages/components/src/Notification/ToastNotification/ToastNotificationsList/ToastNotificationsList.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react"
+import { createPortal } from "react-dom"
+import { useToastNotificationContext } from "../context/ToastNotificationContext"
+import { ToastNotificationsMap } from "./subcomponents/ToastNotificationsMap"
+import styles from "./ToastNotificationsList.module.scss"
+
+export const ToastNotificationsList = (): JSX.Element => {
+  const { notifications, removeToastNotification } =
+    useToastNotificationContext()
+
+  const containers = document.querySelectorAll(
+    '[data-testid="toast-notifications-list"'
+  )
+
+  if (containers) {
+    // Remove any duplicate instances
+    // (eg. Storybook docs page has multiple stories each with their own context)
+    containers.forEach((c, i) => {
+      if (i === 0) return
+      c.remove()
+    })
+  }
+
+  return createPortal(
```

Comment:
Does this mean toasts won't appear on top of modals/inspector/workflow?

Reply:
Hmm...did they ever though? It was injected into `document.body` from the start 🤔

Reply:
Seems the styles include a `z-index`, not high enough for workflow, but seems like toasts should be considered the highest priority over any other floating element. Perhaps this just means we need a token system for `z-index` so we rely on a consistent environment to guarantee that important things are surfaced across the platform.

OOS of course.